### ISSUE TITLE
is there any kml file format and example?

### ISSUE THREAD

**Issue by masayuki-kobayashi1** [AutelSDK/AndroidAdvanceSample#26]
I don't know how to write a kml file that can be read by EVO II Pro V3, I would like to know how to write it or a sample file.
Below are the specific things I want to do.
Interval shooting settings
Gimbal pitch angle settings
Settings for stopping/not stopping the aircraft at each waypoint

memo: Q4

**Comment by AutelSDK** [comment]
We do not support KML. You need to convert the relevant files or your flight data into MissionInfoJNI.

**Comment by masayuki-kobayashi1** [comment]
@AutelSDK 
Autel Explorer has support for KML file import.
You can find it from your site's release note.
https://www.autelrobotics.co.uk/download/113.html
<img width="1474" alt="image" src="https://github.com/AutelSDK/AndroidAdvanceSample/assets/129825536/696c49fb-c91a-49d4-b276-1afaa7945538">

**Comment by masayuki-kobayashi1** [comment]
sdk do not support

**Comment by AutelSDK** [comment]
yes,autel sdk not support kml file,needs you deal it. 无人机应用平台部  冯银华
深圳市道通智能航空技术有限公司
Autel Intelligent Technology Corp., Ltd.  
TEL: +86 18138233315
Email: ***@***.***
Web:http://www.autel.com
深圳市南山区西丽街道学苑大道1001号智园B1栋9层
ADD: 9th Floor, Building B1, Zhiyuan, Xueyuan Road, Xili, Nanshan, Shenzhen,
518055, China-----masayuki kobayashi ***@***.***> 撰写:-----收件人: AutelSDK/AndroidAdvanceSample ***@***.***>发件人: masayuki kobayashi ***@***.***>日期: 2023/09/27 下午02:36抄送: Autel Robotics ***@***.***>, Mention ***@***.***>主题: Re: [AutelSDK/AndroidAdvanceSample] is there any kml file format and example? (Issue #26)
sdk do not support

**Comment by masayuki-kobayashi1** [comment]
@AutelSDK 
I understand that this is a feature provided separately from the SDK.
But, the Autel Exploer app appears to have kml file import capability.
What is the format of the kml file that can be imported by the app and what are some examples?
